app/gui/src/controller/graph/executed.rs

```diff
@@ -375,6 +375,8 @@ impl Handle {
         if self.project.read_only() {
             Err(ReadOnly.into())
         } else {
+            let model = self.project.main_module_model().await?;
+            model.reopen_externally_changed_file().await?;
```

Comment:
Why do we reopen file before restarting execution?

Reply:
In case the file was modified by an external editor, we want to apply those changes

Reply:
It's only an initial part of the issue. When I implement a notification from the language server that the file was modified on disk, the module will be updated by this notification. And not by the keypress.

Reply:
I think if user press "rerun graph" shortcut does not expect reloading changes from disk. I would rather expect running the currently displayed graph.

Reply:
Ok, sounds reasonable. I will assign this action to a different key combination. I will remove it after I implement the notification

Reply:
Assigned this to `cmd+alt+y` keybinding